beacon-chain/utils/spectest/shuffle_test_format.go

```diff
@@ -0,0 +1,20 @@
+package spectest
```

Comment:
This shouldn't be a public target. Lets rename to shuttle_test_format_test.go to keep private to this.